news/0000.breaking

```diff
@@ -0,0 +1 @@
+FIXME: Upgrade to dx site root
```

Comment:
@mauritsvanrees I guess this is something you want to fix before doing the release, right? :)